### ISSUE TITLE
The problem of decreased accuracy on the coco dataset and the dataset with added classes

### ISSUE THREAD

**Issue by penguin345** [ultralytics/ultralytics#7522]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

My detection targets include some classes in coco and other classes , so I selected them from the coco dataset and the Open Images dataset to form my dataset. And use yolov8n pretrained model for training.

However, the accuracy of the trained model decreases significantly compared to the pretrained model, and the recall rate is very low.  Class such as person in the coco dataset is often undetectable. I want to know how to solve this problem, such as from the perspective of datasets, training strategies, or hyperparameters. 

Or if you have any other suggestions, I would greatly appreciate it
